File: 485-max-consecutive-ones/max-consecutive-ones.c
int findMaxConsecutiveOnes(int* nums, int numsSize) {
    int max = 0;
    int cons = 0;
    for(int i = 0;i<numsSize;i++){
        if(nums[i] == 1){
            cons++;
            if (cons>max){
                max = cons;
            }
        }
        else
        cons = 0;
    }
    return max;
    
}

File: 485-max-consecutive-ones/max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        maxi, cons = 0, 0
        for dig in nums:
            cons = cons + 1 if dig == 1 else 0
            maxi = max(cons,maxi)
        return maxi
        